src/main/java/frc/robot/commands/FollowTrajectory.java

```diff
@@ -90,9 +91,13 @@ public FollowTrajectory(DriveTrain driveTrain,
 	// Called when the command is initially scheduled.
 	@Override
 	public void initialize() {
+        if (m_actualTrajectory == null) {
+            m_actualTrajectory = PathPlannerTrajectory.transformTrajectoryForAlliance(m_trajectory, DriverStation.getAlliance());
```

Comment:
Unfortunately, this is going to cause problems. If this is an Auto trajectory, we presumably do want to transform the path. However, we are also using this for on-the-fly trajectories, where we don't want the transform. I guess we should add a boolean to this command?